meta-mender-core/classes/mender-setup-image.inc

```diff
@@ -32,9 +32,13 @@ mender_update_fstab_file() {
     local tmpBootPart="${MENDER_BOOT_PART}"
     local tmpDataPart="${MENDER_DATA_PART}"
     local partuuid="${@bb.utils.contains('MENDER_FEATURES', 'mender-partuuid', 'true', 'false', d)}"
+    local partlabel="${@bb.utils.contains('DISTRO_FEATURES', 'mender-partlabel', 'true', 'false', d)}"
     if [ ${partuuid} = "true" ]; then
         tmpBootPart="PARTUUID=${@mender_get_partuuid_from_device(d, '${MENDER_BOOT_PART}')}"
         tmpDataPart="PARTUUID=${@mender_get_partuuid_from_device(d, '${MENDER_DATA_PART}')}"
+    elif [ ${partlabel} = "true" ]; then
+        tmpBootPart="LABEL=boot"
+        tmpDataPart="LABEL=data"
```

Comment:
I suggest we set these in variables, defaulting to `boot` and `data` respectively. For example:
```suggestion
        tmpBootPart="LABEL=${MENDER_BOOT_PART_LABEL}"
        tmpDataPart="LABEL=${MENDER_DATA_PART_LABEL}"
```

And these being weak set in `mender-setup.bbclass` to "boot" and "data" with a comment explaining that are only used when `mender-partlabel` feature is enabled